app/client/src/widgets/InputWidgetV2/widget/index.tsx

```
@@ -17,7 +17,7 @@ import type { DerivedPropertiesMap } from "WidgetProvider/factory";
 import { ICON_NAMES } from "WidgetProvider/constants";
 import { AutocompleteDataType } from "utils/autocomplete/AutocompleteDataType";
 import BaseInputWidget from "widgets/BaseInputWidget";
-import { isNil, isNumber, merge, toString } from "lodash";
+import { isNil, isNumber, merge, toString, omit } from "lodash";
```

Biome

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)